partition/roles/sonic/templates/sonic-config.yaml.j2

```diff
@@ -0,0 +1,31 @@
+bgp_ports: {{ sonic_bgp_ports }}
+breakouts: {{ sonic_breakouts }}
+docker_routing_config_mode: {{ sonic_docker_routing_config_mode }}
+features: {{ sonic_features }}
+frr_mgmt_framework_config: {{ sonic_frr_mgmt_framework_config }}
+interconnects: {{ sonic_interconnects }}
+hostname: {{ sonic_hostname }}
+lldp_hello_time: {{ sonic_lldp_hello_timer }}
+loopback_address: {{ sonic_loopback_address }}
+mclag: {{ sonic_mclag }}
+mgmt_interface:
+  gateway_address: {{ sonic_mgmtif_gateway }}
+  ip: {{ sonic_mgmtif_ip }}
```

Comment:
How do you ensure that `sonic_mgmtif_gateway` and `sonic_mgmtif_ip` are defined?